ansible_base/jwt_consumer/hub/auth.py

```diff
@@ -1,15 +1,42 @@
+import logging
+
+from django.contrib.contenttypes.models import ContentType
+
 from ansible_base.jwt_consumer.common.auth import JWTAuthentication
 from ansible_base.jwt_consumer.common.exceptions import InvalidService
+from ansible_base.resource_registry.models import Resource
+
+logger = logging.getLogger('ansible_base.jwt_consumer.hub.auth')
 
 
 class HubJWTAuth(JWTAuthentication):
-    def process_permissions(self, user, claims, token):
+    def process_permissions(self):
         # Map teams in the JWT to Automation Hub groups.
         try:
-            from galaxy_ng.app.models.auth import Group
+            from galaxy_ng.app.models import Organization, Team
+
+            self.team_content_type = ContentType.objects.get_for_model(Team)
+            self.org_content_type = ContentType.objects.get_for_model(Organization)
         except ImportError:
             raise InvalidService("automation-hub")
 
-        for team in claims.get("teams", []):
-            hub_group, _ = Group.objects.get_or_create(name=team["name"])
-            hub_group.user_set.add(user)
+        for role_name in self.common_auth.token.get('object_roles', {}).keys():
```

Comment:
It looks like you fixed the issue with references to `self.token` and user? You squashed commits